compression.go

```diff
@@ -44,9 +53,9 @@ type compressedWriter interface {
 	Close() error
 }
 
-type plainReader struct{ io.Reader }
+type readerNoopCloser struct{ io.Reader }
```

Comment:
minor consistency: pls either leave it `plainReader` or rename `plainWriter` too

Reply:
Certainly 👍